packages/clients/diplan/src/store/geoEditing/cutPolygons/index.ts

```diff
@@ -41,11 +43,22 @@ export const cutPolygons = ({
     )
 
     if (cuttables.length) {
-      const cuts = cutCuttablesWithCutter(cuttables, cutter).map(
-        (cut) => converter.readFeature(cut, projectionInfo) as Feature
-      )
-      drawSource.clear()
-      drawSource.addFeatures([...uncuttables, ...cuts])
+      try {
+        const cuts = cutCuttablesWithCutter(cuttables, cutter).map(
+          (cut) => converter.readFeature(cut, projectionInfo) as Feature
+        )
+        drawSource.clear()
+        drawSource.addFeatures([...uncuttables, ...cuts])
+      } catch (e) {
```

Comment:
```suggestion
      } catch {
```
Error is not logged anywhere